### PR TITLE
If email is too long it is aligned incorrectly

### PR DESCRIPTION
**Pull request by dmitrij-borchuk**

### CODE REVIEW

File: dev/app.scss [hunk 1]
```diff
@@ -198,4 +198,10 @@ input {
 .md-raised.md-primary.red-btn {
   background-color: #f44336;
   color: #fff;
-}
\ No newline at end of file
+}
```
Comment (dmitrij-borchuk) [inline dev/app.scss]: Test comment

File: dev/conversation/tpl.pug [hunk 2]
```diff
@@ -18,18 +18,22 @@
...
     md-button.md-primary.md-icon-button.md-ink-ripple.back(ng-click="vm.back($event)")
       i.material-icons arrow_back
-    p.title(ng-if="!vm.conversation.email || vm.conversation.email === null || vm.conversation.email === undefined") Public conversation
-    p.title(ng-if="vm.conversation.email && vm.conversation.email != null && vm.conversation.email != undefined") to:&nbsp
+    .title(ng-if="!vm.conversation.email || vm.conversation.email === null || vm.conversation.email === undefined") Public conversation
+    .title.ellipsis(
+      ng-if="vm.conversation.email && vm.conversation.email != null && vm.conversation.email != undefined"
+      title="{{vm.conversation.email}}"
+    )
+      | to:&nbsp
```
Comment (aleksspeaker) [inline dev/conversation/tpl.pug]: It highlights the `&` symbol. It's probably good to end `&nbsp` with `;`

**Review by aleksspeaker — CHANGES_REQUESTED** [review]

File: dev/conversation/tpl.pug (hunk 2 above)
Comment (dmitrij-borchuk) [reply dev/conversation/tpl.pug]: Good point, I will

File: dev/conversation/styles.scss [hunk 3]
```diff
@@ -7,12 +7,14 @@
 
   //HEADER
   header {
+    display: flex;
```
Comment (dmitrij-borchuk) [inline dev/conversation/styles.scss:10]: For vertical alignment

File: dev/conversation/styles.scss [hunk 4]
```diff
@@ -7,12 +7,14 @@
...
     width: 100%;
     background-color:  #fff;
     box-shadow: 1px 3px 5px 0 rgba(112,157,228,.25);
 
     .logoContainer {
-      display: inline-block;
+      display: flex;
```
Comment (dmitrij-borchuk) [inline dev/conversation/styles.scss:16]: For vertical alignment

**Review by aleksspeaker — APPROVED** [review]

**Pull request merged**